Form/Type/ModelType.php

```diff
@@ -110,6 +110,7 @@ public function configureOptions(OptionsResolver $resolver)
                     $propertyAccessor
                 );
             };
+            $options['choices_as_values'] = true;
```

Comment:
This should probably be conditional, since setting it for sf >= 3.0 is deprecated